docs/architecture-testing.md

```
@@ -106,6 +112,40 @@ archscope validate . --format json --deterministic
+
+Repository scan outputs:
+
+- `.diagram/manifest.json`
+- `.diagram/brief.md`
+- `.diagram/agent-context.json`
+- `.diagram/architecture.mmd`
+
+PR scan outputs:
+
+- all repository scan outputs
+- `.diagram/pr-impact/pr-impact.json` when `--base` or `--head` refs resolve
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)